### PR TITLE
Feature/story test/develop

### PR DESCRIPTION
**Pull request by dumebi**
Changed teller solidity tests to a story mode

view documentation here: https://www.notion.so/Teller-Test-Suite-V2-70cf0306597a41e489b932f0ae1f7b4d

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
     "@types/chai-as-promised": "^7.1.3",
     "@types/expect": "^24.3.0",
     "@types/fs-extra": "^9.0.11",
+    "@types/glob": "^7.1.3",
```
Comment (passabilities) [inline package.json:33]: why did we need to add this?

Comment (passabilities) [inline package.json:33]: @dumebi why did we need to add this?

File: test/helpers/chai-helpers.ts [hunk 2]
```diff
@@ -1,18 +0,0 @@
-import chai from 'chai'
-import { ethers } from 'hardhat'
-
-chai.Assertion.overwriteMethod(
-  'eql',
```
Comment (passabilities) [inline test/helpers/chai-helpers.ts:5]: why did we remove this?

File: test/integration/escrow/dapps/aave.test.ts [hunk 3]
```diff
@@ -14,7 +14,7 @@ chai.use(solidity)
 
 const { getNamedSigner, contracts, evm } = hre
 
-describe('AaveDapp', () => {
+describe.skip('AaveDapp', () => {
```
Comment (mkhoury93) [inline test/integration/escrow/dapps/aave.test.ts:17]: is there a reason we are skipping these tests?

File: test/integration/full.test.ts [hunk 4]
```diff
@@ -2,39 +2,39 @@
 // import { solidity } from 'ethereum-waffle'
 // import { BigNumber, Signer } from 'ethers'
 // import hre from 'hardhat'
-//
+
```
Comment (mkhoury93) [inline test/integration/full.test.ts:5]: we can remove this

**Review by mkhoury93 — DISMISSED** [review]
we need to remove the old test files, since we don't need them given our new tests

File: package.json (hunk 1 above)
Comment (dumebi) [reply package.json:33]: I later didn't need it

File: test/helpers/chai-helpers.ts (hunk 2 above)
Comment (dumebi) [reply test/helpers/chai-helpers.ts:5]: I think this was from the different merges i did

File: test/integration/full.test.ts (hunk 4 above)
Comment (dumebi) [reply test/integration/full.test.ts:5]: okay

**Comment by dumebi** [comment]
> we need to remove the old test files, since we don't need them given our new tests

fixed

**Review by admazzola — DISMISSED** [review]

**Review by passabilities — APPROVED** [review]

**Review by admazzola — APPROVED** [review]

**Pull request merged**